build-logic/jvm/src/main/kotlin/gradlebuild.unittest-and-compile.gradle.kts

```diff
@@ -319,6 +320,10 @@ fun configureTests() {
                 }
             }
         }
+
+        predictiveSelection {
+            enabled.set(project.predictiveTestSelectionEnabled)
```

Comment:
Can you make sure test distribution stays disabled for performance tests? https://github.com/gradle/gradle/pull/20489

Reply:
Enabling PTS won't enable TD but it will change the `TestExecuter` used by the `Test` task. Is that problematic here?

Reply:
Sorry, typo: I meant PTS for the performance test tasks. We don't want to use test selection for the performance test tasks, while we do want to use the TD runner without remote executors.